src/runtime/time.py

```diff
@@ -0,0 +1,8 @@
+from lpython import f64, ccall
+
+def time() -> f64:
+    return _lfortran_time()
```

Comment:
Why not using the `_lfortran_cpu_time` function that is already implemented? It should work and at least it compiles on Windows, even if not implemented yet.